### ISSUE TITLE
request came in to update World Geographical Scheme for Recording Plant Distributions

### ISSUE THREAD

**Issue by pmergen** [tdwg/prior-standards#18]
Dr Marianne le Roux would like to know what the procedure would be to update information in the standard for the World Geographical Scheme for Recording Plant Distributions. would indeed like to update the Standard with information relevant to South Africa and the nine recognised provinces.
(e-Flora Coordinator
South African National Biodiversity Institute (SANBI)
Pretoria National Botanical Gardens
2 Cussonia Avenue, Brummeria, Pretoria
Email: m.leroux@sanbi.org.za
Web: www.sanbi.org)

Help needed to provide them with up to date information and procedure and follow up.

**Comment by peterdesmet** [comment]
The standard is currently the archived zip download, which is not directly editable on GitHub. Now that there is interest in the standard, we should unzip it and check if the file format is readable on GitHub.

From then one, the requested changed can be added. These are all things I can do, but only after the conference.

I'll create a separate issue for potentially moving the standard.

**Comment by pmergen** [comment]
ok thanks

**Comment by peterdesmet** [comment]
Very late response: the standard has now moved to https://github.com/tdwg/wgsrpd, but with no real maintenance team. Might be good to send a call out for folks who want to maintain the standard.

Closing issue here.
